.env.example

```diff
@@ -33,9 +33,9 @@ SENDGRID_SMTP_HOST=smtp.sendgrid.net
 SENDGRID_USER=
 SENDGRID_PW=5ecr3t
 
-ELASTIC_INSTANCE=
-ELASTIC_USER=
-ELASTIC_PASSWORD=
+ELASTIC_HOST=https://example.com:9243
+ELASTIC_ID=5ecr3t
```

Comment:
it's called ELASTIC_USER below

Reply:
Weird, I thought I've updated it. Will fix in 30'

Reply:
fixed